src/main/java/seedu/sudohr/model/SudoHr.java

```diff
@@ -190,8 +190,6 @@ public void addEmployee(Employee employee) {
     public void setEmployee(Employee target, Employee editedEmployee) {
         requireNonNull(editedEmployee);
         employees.setEmployee(target, editedEmployee);
-        refreshDepartments();
```

Comment:
There should be no need to add in logic for edit employee since we are currently just tracking the number of employees in each leaves/department, editing employee has no effect
Though for defensive programming sake, it would be better to refresh it

Reply:
oh right. Normal edits to employee (eg name) will be updated because rendering logic handled diffly.